test/test_linalg.py

```diff
@@ -1337,17 +1337,16 @@ def run_test_case(input, ord, dim, keepdim):
     def test_norm_fused_type_promotion(self, device, dtype):
         x = torch.randn(10, device=device, dtype=dtype)
 
-        def profile_and_check(fn, x, kwargs, fn_name):
+        def profile_and_check(fn, x, kwargs):
             with torch.profiler.profile(activities=(torch.profiler.ProfilerActivity.CPU,)) as p:
                 fn(x, **kwargs, dtype=torch.float)
             # smoke check that profiler returned some events
-            self.assertTrue(fn_name in map(lambda e: e.name, p.events()))
+            self.assertTrue("aten::linalg_vector_norm" in map(lambda e: e.name, p.events()))
```

Comment:
Change to something like:
`self.assertTrue("aten::linalg_vector_norm" in (e.name for e in p.events()))`

(Flagged by flake8 in Meta-internal CI)

Reply:
@seemethere @malfet @peterbell10 it'd be cool to have this rule in OSS (avoid lambdas within maps) to avoid these reverts.

Reply:
I'm looking at bringing closer all Meta-internal and OSS linters. This check in particular should be easy to enable in OSS.

Reply:
Current OSS flake8 is older than Meta-internal flake8.
I have a draft PR to update https://github.com/pytorch/pytorch/pull/84732